src/main/java/org/folio/edge/sip2/utils/Utils.java

```diff
@@ -108,4 +111,30 @@ public static String buildQueryString(Map<String,String> queryStringParameters,
   public static String encode(String url) {
     return URLEncoder.encode(url, StandardCharsets.UTF_8);
   }
+
+  public static IResource handleSearchErrors(Throwable cause, List<String> errorMessages) {
+    IResource temp = new IResource() {
```

Comment:
Can just `return new IResrouce() {` here without assignment. I think sonar recommends this too.